src/internal/transport/websocket/WebSocketClient.ts

```diff
@@ -253,6 +254,18 @@ function register(registrationCallback: () => any, options: WebSocketClientOptio
     });
 }
 
+// Make some attempt to warn them if they are using the wrong team ID.
+// Now, if only the config had provenance and I knew where that was set, so I could tell them how to change it...
+function warnAboutChatTeamId(registrationPayload: any) {
+    if (registrationPayload.team_ids.length === 1) {
+        const singleTeam = registrationPayload.team_ids[0];
+        if (singleTeam.startsWith("T")) {
+            logger.error(`This looks like a Chat Team: ${singleTeam}. You might want your Atomist Team ID, which usually starts with A.` +
+                "Try `@atomist pwd` in Slack to see both.");
+        }
+    }
+}
+
```

Comment:
I don't think these kind of validation messages belong into the client. This should be handled in the API and the client should just display the error message. Also discussed this with @slimslenderslacks.